executor.go

```diff
@@ -60,6 +73,20 @@ var errUndefinedTaskName = errors.New("undefined task name")
 
 type ExecutorOption func(*Executor)
 
+const helpMsgTemplate = `
+"{{ .TaskName }}" supports the following options:
```

Comment:
Didn't sarah tansey add descriptions uh, [example](https://github.com/samsarahq/taskrunner/blob/e95de303736b0a190d6c0269efb3efd0afbedc64/README.md?plain=1#L55-L57). Would be nice to include the task level description in the help message?

Reply:
Oo good call out and ty for link

Reply:
Will now include task description if available
![Screenshot 2023-01-30 at 6 21 57 PM](https://user-images.githubusercontent.com/16676111/215626686-49860dd4-a8a0-409e-9178-12e9242d7c48.png)